### PR TITLE
Close missing brackets in log texts

### PR DESCRIPTION
**Pull request by shrmnk**
Minor fixes in logging texts for the workbooks_endpoint.py file

### CODE REVIEW

File: tableauserverclient/server/endpoint/workbooks_endpoint.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def update_connection(self, workbook_item, connection_item):
         server_response = self.put_request(url, update_req)
         connection = ConnectionItem.from_response(server_response.content, self.parent_srv.namespace)[0]
 
-        logger.info('Updated workbook item (ID: {0} & connection item {1}'.format(workbook_item.id,
+        logger.info('Updated workbook item (ID: {0} & connection item {1})'.format(workbook_item.id,
                                                                                   connection_item.id))
```
Comment (t8y8) [inline tableauserverclient/server/endpoint/workbooks_endpoint.py]: Looks like the linter is complaining here, the `c` should align with the `w` (one more space in front)

**Comment by t8y8** [comment]
@shrmnk Hey! Are you still interested in updating the patch and merging this?

**Comment by shrmnk** [comment]
Hey sorry missed out on this. Fixed!

**Comment by t8y8** [comment]
@shrmnk Looks good! But, as I was about to merge, I noticed this is against master, can you rebase it against `development`? I'll hit merge as soon as I see that come through.

**Comment by shrmnk** [comment]
Rebased to `development`, thanks!

**Comment by t8y8** [comment]
It looks like it's still showing as a PR trying to merge in to 'master'

**Comment by jacalata** [comment]
@t8y8 check I didn't break anything?

**Comment by t8y8** [comment]
WE CAN CHANGE THE TARGET BRANCH?!?! I never knew!

**Pull request merged**